lib/Notes/NotePopupModal/NotePopupModal.js

```diff
@@ -114,13 +116,13 @@ const NotePopupModal = ({
   }, [mutator.popupNote, getPopupNotesFromNotes]);
 
   useEffect(() => {
-    if (entityId && !isNotesLoading.current) {
+    if (entityId && entityId !== prevEntityId && !isNotesLoading.current) {
       mutator.NotePopupModal_assignedNotes.GET()
         .then(handleGetNotesRequest);
 
       isNotesLoading.current = true;
     }
-  }, [entityId, handleGetNotesRequest, mutator.NotePopupModal_assignedNotes]);
+  }, [entityId, handleGetNotesRequest, mutator.NotePopupModal_assignedNotes, prevEntityId]);
```

Comment:
I think we can fix it by removing `mutator.NotePopupModal_assignedNotes` from deps. It changes every render and triggers this effect